hyperwallet/models.py

```diff
@@ -70,6 +70,8 @@ class User(HyperwalletModel):
         A dictionary containing the attributes for the User.
     '''
 
+    filters_array = {'clientUserId', 'email', 'programToken', 'status', 'verificationStatus'}
```

Comment:
Update the comment from dictionary to set. This is a set in Python. Not an array or dictionary. Do for all the similar comments